src/Akka.Persistence.SqlServer/Akka.Persistence.SqlServer.csproj

```diff
@@ -29,9 +29,15 @@ ALTER TABLE {your_snapshot_table_name} ADD COLUMN SerializerId INTEGER NULL
     <EmbeddedResource Include="sql-server.conf" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Akka" Version="1.3.2" />
-    <PackageReference Include="Akka.Persistence" Version="1.3.2" />
-    <PackageReference Include="Akka.Persistence.Sql.Common" Version="1.3.2" />
+    <PackageReference Include="Akka" Version="1.3.7" />
+    <PackageReference Include="Akka.Persistence" Version="1.3.7" />
+    <PackageReference Include="Akka.Persistence.Sql.Common" Version="1.3.7" />
+    <PackageReference Include="Akka.TestKit" Version="1.3.7" />
```

Comment:
Is this needed?

Reply:
Seems to be reverting https://github.com/akkadotnet/Akka.Persistence.SqlServer/pull/93

Reply:
@ondrwjpialek I would guess not I’m guessing the nuget package manager auto-added those when trying to update to update the other references for some reason, you are right unless it’s a test project it doesn’t make sense to have a dependency on testkit’s, I’ll remove those and update the pull request in the morning when I’m back in the office